scripts/pkg/scripts/pre_install.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/sh
+
+# SPDX-License-Identifier: Apache-2.0
+#
+# The OpenSearch Contributors require contributions made to
+# this file be licensed under the Apache-2.0 license or a
+# compatible open source license.
+
+# Description:
+# Pre-install script to stop "<%= product %>".service if it's running.
+
+set -e
+
+echo start pre install
+
+if command -v systemctl >/dev/null && systemctl is-active "<%= product %>".service >/dev/null; then
+    systemctl --no-reload stop "<%= product %>".service
```

Comment:
Add some echo so we can see what's going on.

Reply:
Added.